packaging/helm/skipper-aws/values.yaml

```diff
@@ -0,0 +1,33 @@
+clusterID: mycluster
+
+# AWS IAM integration kube2iam or kube-aws-iam-controller
+install_kube2iam: true
```

Comment:
This is my personal preference but I tend to like nested yaml for this sort of thing, e.g.

```yaml
kube2iam:
  enabled: true
  role: kube2iam-role
  etc...
```

The Helm best practice docs don't suggest one way over another so don't feel like you need to change this.

Reply:
makes sense, will change it

Reply:
fixed